src/incflo_apply_corrector.cpp

```diff
@@ -156,325 +143,26 @@ void incflo::ApplyCorrector()
                        get_density_new_const(), GetVecOfConstPtrs(vel_eta));
     }
 
-    if (m_advect_tracer && m_diff_type == DiffusionType::Explicit) {
-        compute_laps(get_laps_new(), get_tracer_new_const(),
```

Comment:
My understanding is that this has been moved to `update_tracer`, which is called after `update_density` updates the new density. So the density used by compute_laps has changed in this PR. Is that intentional?

Reply:
density isn't actually used in compute_laps -- I'll remove that from the argument list.